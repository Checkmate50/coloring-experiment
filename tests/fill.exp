a : A;
b : B;
c : C;
d : D;
e : E;

e < d;
d < c;
e < b;
d < b;
c < a;

???;