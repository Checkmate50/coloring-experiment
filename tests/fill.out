e;
d;
b;
c;
a;
